public/app/core/angular_wrappers.ts

```diff
@@ -109,4 +115,10 @@ export function registerAngularDirectives() {
     'onChange',
     ['datasource', { watchDepth: 'reference' }],
   ]);
+  react2AngularDirective('datasourceHttpSettingsNext', DatasourceHttpSettings, [
```

Comment:
we need something that is backward compatible with the old directive as well.

Reply:
@torkelo for the backward compatibility I have not removed the old directive. Instead, it's using the new one under the hood https://github.com/grafana/grafana/pull/19452/files#diff-11d01ebf4e4e29f39faeca6e1f9273f6R4

Reply:
yes, that's great. I missed that